stable/doc_hardware.md

```diff
@@ -1 +1,107 @@
-coming soon...
\ No newline at end of file
+# Hardware documentation
+
+In this section we present a short documentation of our kit. More informations about the hardware can be found in our [website](http://echopen.org/doc-website/CAT-configuration/CFG-sweeping_probe/content.html) or download our [github folder](https://github.com/echopen/PRJ-medtec_kit), aquarium configuration (PRJ-medtec_kit/configuration/CFG-aquarium_kit/readme.md). We show synoptic and block diagrams of the kit and give some informations about daughter boards and modules.
+
+### Synoptic diagram
+
+Here is a simple diagram presenting the general functions of the device.
+
+![](./doc_hardware/synoptic_diagram2.png)
+
+
+### Block diagram
+
+Here one can find the detailled block diagram of the device to know exactly what is done by the hardware to make acoustic imaging.
+
+![](./doc_hardware/block_diagram.png)
+
+The device generate a high voltage pulse to excite the transducer (*via* or not an impedance matching layer). The rest of the device is protected from the high voltage pulse by a T/R switch. The signal is amplified by a TGC (variable amplification driven by the controller), filtered (pass-band filter) and converted to digital signal. The signal is the send *via* wifi to a terminal for display.
+
+For making a 2D image, the transducer must make a sweeping movement. The transducer is placed on the choosen line by the controller, acquisition is made and send to the terminal, then the transducer is moved to the next line. The lines are angulary equally reparted on the sector of measurement.
+
+## Device informations
+
+### Acoustic
```

Comment:
Can you move the "### Acoustic" section to characteristics.md, in a dedicated "Transducer characteristics" section ?